app/src/main/java/com/vultisig/wallet/ui/models/peer/PeerDiscoveryViewModel.kt

```
@@ -480,14 +524,15 @@ internal class PeerDiscoveryViewModel @Inject constructor(
     }
+
     private suspend fun isSessionStarted(): Boolean {
-        try {
-            return sessionApi.getParticipants(
+        return try {
+            sessionApi.getParticipants(
                 serverUrl,
                 sessionId
-            ).size > 0
+            ).isNotEmpty()
         } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)